src/kartograf/tests/test_ring_filters.py

```diff
@@ -85,3 +85,26 @@ def test_whole_rings_safe():
     newmapping = filters.filter_whole_rings_only(m1, m2, mapping)
 
     assert newmapping == mapping
+
+
+@pytest.mark.parametrize('molA_B_exp_mapping', [
+    (Chem.MolFromSmiles("C1CCCC2C1CCCC2"), Chem.MolFromSmiles(
```

Comment:
the line breaks here to fit into 80 char lines is making the code worse to read.  If you wrap the lines like below, it's easy to compare the SMILES

```
    (Chem.MolFromSmiles("C1CCCC2C1CCCC2"),
     Chem.MolFromSmiles("C1CCCC2C1CCCC2"),
     {i:i for i in range(10)}),
```